torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6523,11 +6553,6 @@ def method_tests():
         ('resize_as_', (), (non_differentiable(torch.tensor(5.)),), 'scalar'),
         ('resize_as_', (), (non_differentiable(torch.randn((1, 1, 1))),), 'scalar_to_dims'),
         ('resize_as_', (S, S, S), (non_differentiable(torch.randn(S * S, S)),)),
-        ('where', (M, M), (mask_not_all_zeros((M, M)), (M, M)), '', (True,)),
-        ('where', (M, 1, M), (mask_not_all_zeros((M, M)), (M, M, 1)), 'broadcast_all', (True,)),
-        ('where', (), (bernoulli_scalar(), ()), 'scalar', (True,)),
-        ('where', (M, 1, M), (bernoulli_scalar(), (M, M, 1)), 'scalar_broadcast_mask', (True,)),
```

Comment:
Follow-up TODO: delete the bernoulli_scalar() function, it's only used in one sample input and can be replaced with make_tensor(). Also I think it's incorrectly implemented?

Reply:
Why incorrectly?
```python
>>> torch.tensor(0, dtype=torch.bool).bernoulli_()
tensor(True)
>>> torch.tensor(0, dtype=torch.bool).bernoulli_()
tensor(False)
>>> torch.tensor(0, dtype=torch.bool).bernoulli_()
tensor(False)
>>> torch.tensor(0, dtype=torch.bool).bernoulli_()
tensor(False)
>>> torch.tensor(0, dtype=torch.bool).bernoulli_()
tensor(True)
>>> torch.tensor(0, dtype=torch.bool).bernoulli_()
tensor(False)
```

Reply:
Aha. You're right, because torch.bernoulli() and torch.tensor.bernoulli_() are actually different functions, which is a frustrating UX consistency

Actually, would you file an issue for this? 

Docs for bernoulli_():

https://pytorch.org/docs/master/generated/torch.Tensor.bernoulli_.html?highlight=bernoulli_#torch.Tensor.bernoulli_

Docs for bernoulli():

https://pytorch.org/docs/master/generated/torch.bernoulli.html?highlight=bernoulli#torch.bernoulli

The function takes the likelihood from the tensor while the inplace variant takes the input from its `p` input

Reply:
Sure!